.github/workflows/docsbuild-release.yaml

```
@@ -0,0 +1,78 @@
+name: ansible-release-porting-guide
```

The workflow names should normally be text. You can use identifiers like this in the YAML file name if needed.

This should improve the UI representation:
```suggestion
name: Create an Ansible Release Porting Guide PR
```